lib/livebook_web/live/file_select_component.ex

```diff
@@ -268,11 +268,14 @@ defmodule LivebookWeb.FileSelectComponent do
       <:toggle>
         <button
           type="button"
-          class="button-base button-gray button-square-icon"
+          class="button-base button-gray pl-3 pr-2"
           aria-label="switch file system"
           disabled={@file_system_select_disabled}
         >
-          <.file_system_icon file_system={@file.file_system} />
+          <span><%= file_system_name(@file.file_system) %></span>
+          <div class="pl-0.5 flex items-center">
```

Comment:
Should we include the separator like we do for Code cells for consistency?

Reply:
I realised they are different, with Code cells these are effectively two buttons, left part to insert the cell and the right part to show the menu. In this case it's a single action so I went with no separator :)

Reply:
Ah, good call!